src/main/java/com/google/devtools/build/lib/sandbox/WindowsSandboxUtil.java

```diff
@@ -0,0 +1,244 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+//
```

Comment:
Instead of copying `LinuxSandboxUtil`, can you reuse that code?
Maybe you can encapsulate the Linux- and Windows-specific things in a strategy objects? ("Strategy" design pattern).

Reply:
> Instead of copying LinuxSandboxUtil, can you reuse that code?

This is not really a copy of `LinuxSandboxUtil`. `isAvailable` and `waitForProcess` method comes from `sandboxfs`, `CommandLineBuilder` inner class is similar to `LinuxSandboxUtil`'s version, but windows-sandbox doesn't have certain flags that linux-sandbox does.

Reply:
Thanks for clarifying!